models/social_connection.py

```diff
@@ -0,0 +1,69 @@
+import json
```

Comment:
I'm not sure, I was just playing devil's advocate.

One way to decide would be to think about how we would query them. I'm a fan of the JSON property because I don't think we would ever query for anything other than "get me all social connections for team XXX." e.g. we would never do "get me all Facebook connections for event XXX (which would us to make an intermediate query for Teams anyways)." Also, unlike Robots, the number of social connections doesn't increase over time for one Team.